c/planarity.c

```diff
@@ -19,12 +19,156 @@ void ProjectTitle()
 	// Furthermore, a change of Major, Minor or Maintenance here should cause a change
 	// of Current, Revision and/or Age as documented in configure.ac
 
-    Message("\n=================================================="
-            "\nThe Edge Addition Planarity Suite version 3.0.2.0"
-            "\nCopyright (c) 1997-2022 by John M. Boyer"
-    		"\nContact info: jboyer at acm.org"
-            "\n=================================================="
-            "\n");
+	return "\n=================================================="

```

Comment:
If you remove the equal sign lines from this string and instead place them where you currently use this string, then the smaller resulting project title can be used twice to clean up a code duplication.